package.json

```diff
@@ -44,7 +44,6 @@
     "vs:publish:ci": "vsce publish --yarn --pat $VSCODE_STORE_TOKEN",
     "ov:publish:ci": "ovsx publish --yarn --pat $OPEN_VSX_STORE_TOKEN",
     "node:package": "webpack --mode production",
-    "web:package": "webpack --mode production",
```

Comment:
@evoactivity I think we still need it :)

Reply:
this was removed by mistake, I'll put it back